doc/modules/mod_event_pusher_http.md

```diff
@@ -110,10 +110,23 @@ Below is an example of what the body of an HTTP POST request can look like:
 
 If you'd like to learn more about metrics in MongooseIM, please visit [MongooseIM metrics](../operation-and-maintenance/MongooseIM-metrics.md) page.
 
-| Name | Type | Description (when it gets incremented) |
-| ---- | ---- | -------------------------------------- |
-| `[Host, mod_event_pusher_http, sent]` | spiral | An HTTP notification is sent successfully. |
-| `[Host, mod_event_pusher_http, failed]` | spiral | An HTTP notification failed. |
-| `[Host, mod_event_pusher_http, response_time]` | histogram | Does not include timings of failed requests. |
+Prometheus metrics have a `host_type` label associated with these metrics.
+Since Exometer doesn't support labels, the host types, or word `global`, are part of the metric names, depending on the [`instrumentation.exometer.all_metrics_are_global`](../configuration/instrumentation.md#instrumentationexometerall_metrics_are_global) option.
+
+=== "Prometheus"
+
+    | Name | Type | Description (when it gets incremented) |
+    | ---- | ---- | -------------------------------------- |
+    | `mod_event_pusher_http_sent_count` | counter | An HTTP notification is sent successfully. |
+    | `mod_event_pusher_http_sent_failure_count` | counter | An HTTP notification failed. |
+    | `mod_event_pusher_http_sent_response_time` | histogram | Does not include timings of failed requests. |
```

Comment:
I think it would be a good idea to first describe what the metric does and then add the comment about failed requests
```suggestion
    | `mod_event_pusher_http_sent_response_time` | histogram | Time taken to send HTTP notification. Does not include timings of failed requests. |
```

Reply:
I agree. I must admit, with module metrics, I mostly copied the old description.